# The MSVC Resource Compiler truncates macro names at char 31. Therefore all BUILDFLAG ifdefs needs to be patched manually.

--- a/build/build_config.h
+++ b/build/build_config.h
@@ -46,7 +46,7 @@
 //    ARCH_CPU_31_BITS / ARCH_CPU_32_BITS / ARCH_CPU_64_BITS
 //    ARCH_CPU_BIG_ENDIAN / ARCH_CPU_LITTLE_ENDIAN
 
-#ifndef BUILD_BUILD_CONFIG_H_
+#if !defined(BUILD_BUILD_CONFIG_H_) && !defined(RC_INVOKED)
 #define BUILD_BUILD_CONFIG_H_
 
 #include "build/buildflag.h"
--- a/chrome/app/chrome_command_ids.h
+++ b/chrome/app/chrome_command_ids.h
@@ -378,7 +378,7 @@
 #define IDC_MEDIA_ROUTER_ABOUT 51200
 #define IDC_MEDIA_ROUTER_HELP 51201
 #define IDC_MEDIA_ROUTER_LEARN_MORE 51202
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if (0)
 #define IDC_MEDIA_ROUTER_REPORT_ISSUE 51203
 #endif
 #define IDC_MEDIA_ROUTER_ALWAYS_SHOW_TOOLBAR_ACTION 51204
@@ -387,7 +387,7 @@
 #define IDC_MEDIA_ROUTER_TOGGLE_MEDIA_REMOTING 51208
 
 // Context menu items for media toolbar button
-#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
+#if (0)
 #define IDC_MEDIA_TOOLBAR_CONTEXT_REPORT_CAST_ISSUE 51209
 #endif
 #define IDC_MEDIA_TOOLBAR_CONTEXT_SHOW_OTHER_SESSIONS 51210
@@ -421,7 +421,7 @@
 #define IDC_CONTENT_CONTEXT_ACCESSIBILITY_LABELS 52411
 #define IDC_CONTENT_CONTEXT_ACCESSIBILITY_LABELS_TOGGLE_ONCE 52412
 
-#if BUILDFLAG(IS_LINUX)
+#if (0)
 // Screen AI
 #define IDC_RUN_SCREEN_AI 52420
 #endif
